### ISSUE TITLE
Email Notifications: Delivery Options

### ISSUE THREAD

**Issue by carinapilar** [Automattic/wp-calypso#56438]
### What

Add delivery options to WordPress.com emails.

### Why

Under [Reader > Manage](https://wordpress.com/following/manage), and [Notifications > Reader Subscriptions](https://wordpress.com/me/notifications/subscriptions), users are able to set the frequency for email notifications, but this isn't available for WordPress.com emails, set on [Notifications > Updates](https://wordpress.com/me/notifications/updates).

It would be nice to give users the option to get a daily or weekly digest of those emails.

### How

This was originally requested here: pd4QKa-ho-p2

**Comment by inaikem** [comment]
Excellent suggestion! Moving to the `Subscriptions Pod` inbox for consideration there.

@kwight, it's a long shot given everything else in the pod right now but do you think we can squeeze this in with the email template updates?

**Comment by kwight** [comment]
True, good idea, but I think it'll be for another time – adding to the backlog.
